### ISSUE TITLE
Center of Pressure Task

### ISSUE THREAD

**Issue by andreadelprete** [stack-of-tasks/tsid#125]
I'm currently working on a new center of pressure (CoP) task for TSID. Currently we have a center of mass task and an angular momentum task, but no CoP task. This has been requested by @NahuelVilla and it could probably be useful for anybody working with a Linear Inverted Pendulum model. I open this issue to keep track of the developments and advertise this new feature.

**Comment by andreadelprete** [comment]
I've just finished the first successful test of the CoP task. Starting from the `ex_4_walking.py` script (from the exercise folder) I've added a CoP task to track the reference CoP (besides tracking the reference CoM). Before introducing the task, this was the CoP I was getting:
![cop_wo_task](https://user-images.githubusercontent.com/1797690/110591622-9671d300-8179-11eb-9d0f-b694dc55439b.png)

Now with the CoP task it seems that the real CoP is much closer to the reference one:
![cop_w_task](https://user-images.githubusercontent.com/1797690/110591488-6c201580-8179-11eb-884e-c9fd0eaae027.png)

I cannot get the tracking to be better than this because if I increase too much the weight of the CoP task then the robot falls. This is reasonable because I am setting an open-loop reference CoP, which is not stabilizing.
